packages/ui/src/Badge.tsx

```diff
@@ -1,23 +1,31 @@
 import React, {useContext} from "react";
-import {View} from "react-native";
+import {Text, View} from "react-native";
 
 import {BadgeProps, SurfaceTheme, TextTheme} from "./Common";
 import {Icon} from "./Icon";
-import {Text} from "./Text";
 import {ThemeContext} from "./Theme";
 
 export const Badge = ({
   value,
-  iconName = "check",
+  iconName,
   status = "info",
   secondary = false,
-  variant = "text",
+  variant,
   maxValue = 100,
 }: BadgeProps): React.ReactElement => {
   const {theme} = useContext(ThemeContext);
+  const isIconOnly = variant === "iconOnly";
 
   let badgeColor: keyof TextTheme = "inverted";
 
+  const secondaryBorderColors = {
```

Comment:
No way to get these from the theme?

Reply:
Looks like they're not theme colors in Figma, except that doesn't make sense if someone's going to change the theme. I'll add a todo (and I'll chat with Jo)